src/auth/auth.controller.ts

```diff
@@ -0,0 +1,19 @@
+import { AuthService } from './auth.service';
+import { CadastroRequestBody } from './dto/cadastro-Request-Body.dto';
+import { Controller, Post, Body, HttpCode, HttpStatus } from '@nestjs/common';
+import {LoginRequestBody } from './dto/login-Request-Body.dto';
+import { Public } from './decorators/isPublic.decorator';
+
+
+
+@Controller('auth')
+export class AuthController {
+  constructor(private readonly authService: AuthService) {}
+  
+  @Public()
+  @HttpCode(HttpStatus.OK)
+  @Post('login')
+  login(@Body() loginRequestBody : LoginRequestBody){
+      return this.authService.login(loginRequestBody)
+  }
+}
```

Comment:
Considere criar rota para cadastro:

`@Public() 
@HttpCode(HttpStatus.CREATED) 
@Post('signup') 
signup(@Body() cadastroRequestBody: CadastroRequestBody) { 
     return this.authService.cadastro(cadastroRequestBody); 
     }`
     
Ao invés de criar outro DTO chamado cadastroRequestBody, você pode simplesmente importar o DTO de criar usuário constante em seu respectivo módulo.